### PR TITLE
Several links were broken, now fixed, and cleaned up parts 1 and 2 of…

### PR DESCRIPTION
**Pull request by reshmakh**
… tutorials.  Added Github links

### CODE REVIEW

File: packages/docs/docs/auth/access-control.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 sidebar_position: 4
 ---
 
-# Access Controls
+# Access Policies
```
Comment (codyebberson) [inline packages/docs/docs/auth/access-control.md:5]: The original title "Access Controls" was intentional -- I think of "Access Controls" as a general concept, and "AccessPolicy" as a Medplum-specific concept.  They are different, and this page is about Access Controls more generally.

Sorry if that's too pedantic

I'd also prefer that we keep (1) titles, (2) filenames, and (3) URL slugs all consistent.

File: packages/docs/docs/charting/index.md [hunk 2]
```diff
@@ -29,3 +29,7 @@ Some implementations have a library of [Questionnaires](/docs/questionnaires/) t
 ## Placing Orders
 
 Placing orders requires constructing the right resources, for example [CarePlans](/docs/api/fhir/resources/careplan.mdx) and [MedicationRequests](/docs/api/fhir/resources/medicationrequest.mdx) or others. Similar to notes, ordering workflows can be done by creating resources directly, or using [Questionnaires](/docs/questionnaires/) that practitioners fill out and use [Bots](/docs/bots/) to drive workflow and integrations.
+
+## Reference
+
+- [Charting Features and Fixes](https://github.com/medplum/medplum/pulls?q=is%3Apr+label%3Acharting+) on Github
```
Comment (codyebberson) [inline packages/docs/docs/charting/index.md:35]: Is this URL correct?  Trailing "+" seems odd.

File: packages/docs/docs/compliance/index.md [hunk 3]
```diff
@@ -29,3 +29,8 @@ The materials in this section can be used in multiple contexts, and depend on yo
 - When engaging a customer or partner, you may need to answer their security questionnaire or vendor qualification process. Materials here can help.
 - When pursuing certification, for example SOC2, or CLIA/CAP Accreditation you may use materials here and submit them as part of your process.
 - To eligible for use in a clinical or research setting, you may need to demonstrate certain capabilities or conformance to market or sell a product.
+
+## Reference
+
+- [Medplum Compliance](https://github.com/medplum/medplum/pulls?q=is%3Apr+label%3Acompliance) on Github
+- [Medplum Audit and Logging](https://github.com/medplum/medplum/pulls?q=is%3Apr+label%3Aaudit-logging+) on Github
```
Comment (codyebberson) [inline packages/docs/docs/compliance/index.md:36]: Trailing "+"

**Review by codyebberson — CHANGES_REQUESTED** [review]
Overall excellent

2 questions in the comments:

1) "Access Controls" vs "Access Policy"
2) Check trailing "+" characters in URLs

File: packages/docs/docs/tutorials/hello-world-part-1.md [hunk 4]
```diff
@@ -3,59 +3,30 @@ sidebar_position: 4
 sidebar_label: Hello World App
 ---
 
-# Hello World Part 1
+# Hello World Part
```
Comment (rahul1) [inline packages/docs/docs/tutorials/hello-world-part-1.md:6]: If we're dropping the `1`, shouldn't we just call this "Hello World"?

File: packages/docs/src/pages/pricing.tsx [hunk 5]
```diff
@@ -337,12 +337,12 @@ export default function PricingPage(): JSX.Element {
               </li>
               <li id="note7">
                 Data usage refers to the creation of{' '}
-                <a href="https://www.medplum.com/docs/tutorials/api-basics/create-fhir-data">FHIR Resources</a>. This
-                figure is cumulative.
+                <a href="https://www.medplum.com/docs/fhir-datastore/create-fhir-data">FHIR Resources</a>. This figure
```
Comment (rahul1) [inline packages/docs/src/pages/pricing.tsx:340]: I think this should be a relative link, no?

File: packages/docs/src/pages/pricing.tsx [hunk 6]
```diff
@@ -337,12 +337,12 @@ export default function PricingPage(): JSX.Element {
...
+                is cumulative.
               </li>
               <li id="note8">
                 Bots and automation refer to custom logic written by customers to execute their workflow.{' '}
-                <a href="https://www.medplum.com/products/automation">Automation</a> documentation and{' '}
+                <a href="https://www.medplum.com/products/bots">Automation</a> documentation and{' '}
```
Comment (rahul1) [inline packages/docs/src/pages/pricing.tsx:345]: relative link?

File: packages/docs/docs/auth/access-control.md (hunk 1 above)
Comment (reshmakh) [reply packages/docs/docs/auth/access-control.md:5]: Got it - yeah, let's use general concepts.  At a high level these should have a mental "How To" in the front, so I'm going to rename to "Enable Access Controls"

**Review by rahul1 — APPROVED** [review]

File: packages/docs/docs/charting/index.md (hunk 2 above)
Comment (reshmakh) [reply packages/docs/docs/charting/index.md:35]: These work - both with and without trailing +, will clean it up.

File: packages/docs/docs/tutorials/hello-world-part-1.md (hunk 4 above)
Comment (reshmakh) [reply packages/docs/docs/tutorials/hello-world-part-1.md:6]: Yeah, we should

File: packages/docs/src/pages/pricing.tsx (hunk 5 above)
Comment (reshmakh) [reply packages/docs/src/pages/pricing.tsx:340]: Relative doesn't work in the JSX elements

File: packages/docs/src/pages/pricing.tsx (hunk 6 above)
Comment (reshmakh) [reply packages/docs/src/pages/pricing.tsx:345]: Doesn't work in the JSX elements

**Review by codyebberson — APPROVED** [review]

**Pull request merged**